node/src/chain_spec.rs

```diff
@@ -203,6 +203,52 @@ pub fn get_chain_spec(para_id: ParaId) -> ChainSpec {
 	)
 }
 
+/// Generate testing chain_spec for staking integration tests with accounts initialized for
+/// collating and nominating.
+#[cfg(feature = "test-staking")]
+pub fn staking_spec(para_id: ParaId) -> ChainSpec {
+	ChainSpec::from_genesis(
+		"Moonbase Development Testnet",
+		"staking",
+		ChainType::Local,
+		move || {
+			testnet_genesis(
```

Comment:
Would be nice to have a comment explaining what each line is for so that I can modify it myself. Does this allow changing the config of every pallet we have, like the specs did?

Reply:
It's calling the function a few lines below called `testnet_genesis` so you can't understand the config without looking at what that function does with it. I added some comments to improve readability though.

Reply:
It provides _almost_ as much configurability as the json spec. Instead of setting the balances for accounts, it mints `1 << 80` balance for the specified `endowed_accounts: Vec<AccountId>`.

Reply:
> It provides almost as much configurability

If the current approach doesn't suit your needs, we can change that `testnet_genesis` function or remove it, or whatever makes it most expressive.